salt/roster/flat.py

```diff
@@ -44,19 +44,20 @@ def targets(tgt, tgt_type='glob', **kwargs):
     conditioned_raw = {}
     for minion in raw:
         conditioned_raw[str(minion)] = raw[minion]
-    rmatcher = RosterMatcher(conditioned_raw, tgt, tgt_type, 'ipv4')
+    rmatcher = RosterMatcher(conditioned_raw, tgt, tgt_type, 'ipv4', opts=__opts__)
     return rmatcher.targets()
 
 
 class RosterMatcher(object):
     '''
     Matcher for the roster data structure
     '''
-    def __init__(self, raw, tgt, tgt_type, ipv='ipv4'):
+    def __init__(self, raw, tgt, tgt_type, ipv='ipv4', **kwargs):
         self.tgt = tgt
         self.tgt_type = tgt_type
         self.raw = raw
         self.ipv = ipv
+        self.opts = kwargs.get('opts', {})
```

Comment:
IMO, like with pretty much everything else, the __opts__ should always be there, because it should only be loaded from the loader.